src/sqlfluff/core/default_config.cfg

```diff
@@ -281,7 +281,7 @@ require_final_semicolon = False
 # GROUP BY/ORDER BY column references
 group_by_and_order_by_style = consistent
 
-[sqlfluff:rules:L057]
+[sqlfluff:rules:RF05]
```

Comment:
Are there any plans to allow using the new "readable" rule names in the config rather than the codes?

Reply:
I'll slack you about this. Short answer - not at the moment because if we accept more than one way of specifying config, we then have to deal with resolving priority.